app/src/main/java/ca/bc/gov/bchealth/ui/login/BcscAuthViewModel.kt

```diff
@@ -459,7 +459,8 @@ data class AuthStatus(
 
 enum class LoginStatus {
     ACTIVE,
-    EXPIRED
+    EXPIRED,
+    UN_AUTHENTICATED
```

Comment:
Could you rename to either **UNAUTHENTICATED** or **NOT_AUTHENTICATED** ?

Reply:
done